modules/installation-configuration-parameters.adoc

```diff
@@ -210,6 +210,15 @@ accounts for the dramatically decreased machine performance.
 |The number of control plane machines to provision.
 |A positive integer greater than or equal to `3`. The default value is `3`.
 
+|`credentialsMode`
+|The Cloud Credential Operator (CCO) mode. If no mode is specified, the CCO defaults to running in mint mode.
+[NOTE]
+====
+Not all CCO modes are supported for all cloud providers. For more information on 
+CCO modes, see the xref:../../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[Cloud Credential Operator] page.
+====
+|`Mint`, `Passthrough`, or `Manual`.
```

Comment:
Also valid to have the default empty string "" (for CCO to dynamically determine the credentials capabilities).